relate/src/main/scala/com/lucidchart/open/relate/SqlRow.scala

```diff
@@ -0,0 +1,349 @@
+package com.lucidchart.open.relate
+
+import java.io.InputStream
+import java.io.Reader
+import java.net.URL
+import java.nio.ByteBuffer
+import java.sql.Blob
+import java.sql.Clob
+import java.sql.NClob
+import java.sql.Ref
+import java.sql.ResultSetMetaData
+import java.sql.RowId
+import java.sql.SQLException
+import java.sql.SQLXML
+import java.sql.Time
+import java.sql.Timestamp
+import java.util.Calendar
+import java.util.Date
+import java.util.UUID
+import scala.collection.JavaConversions
+import scala.collection.generic.CanBuildFrom
+import scala.collection.mutable
+import scala.collection.mutable.Builder
+import scala.collection.mutable.MutableList
+import scala.util.Try
+
+object SqlRow {
+  def apply(rs: java.sql.ResultSet): SqlRow = new SqlRow(rs)
+}
+
+class SqlRow(val resultSet: java.sql.ResultSet) extends ResultSetWrapper {
+  /**
+    * Get the number of the row the SqlResult is currently on
+    * @return the current row number
+    */
+  def getRow(): Int = resultSet.getRow()
+
+  def strictArray(column: String): java.sql.Array = resultSet.getArray(column)
```

Comment:
well honestly, I'm more inclined to remove the non-strict versions and replace them with the strict versions.